File: switch case.php
<?php
// ATM Sederhana - PHP

// Validasi Nama
do {
    echo "Masukkan Nama: ";
    $nama = trim(fgets(STDIN));

    echo "Apakah nama sudah benar? (TRUE/FALSE): ";
    $valid = trim(fgets(STDIN));
} while (strtoupper($valid) !== "TRUE");

// Input NIM dan Konversi ke Saldo
echo "Masukkan NIM (angka saja): ";
$nim = trim(fgets(STDIN));
$saldo = (int)$nim;

// Tampilkan data awal
echo "\n=== DATA NASABAH ===\n";
echo "Nama  : $nama\n";
echo "Saldo : Rp " . number_format($saldo, 0, ',', '.') . "\n\n";

// Menu ATM
while (true) {
    echo "===== MENU ATM =====\n";
    echo "1. Cek Saldo\n";
    echo "2. Tarik Tunai\n";
    echo "3. Setor Tunai\n";
    echo "4. Transfer\n";
    echo "5. Keluar\n";
    echo "Pilih menu: ";
    $pilih = trim(fgets(STDIN));

    switch ($pilih) {
        case 1:
            echo "Saldo Anda: Rp " . number_format($saldo, 0, ',', '.') . "\n\n";
            break;

        case 2:
            echo "Jumlah tarik: ";
            $tarik = trim(fgets(STDIN));
            if ($tarik <= $saldo) {
                $saldo -= $tarik;
                echo "Tarik berhasil. Sisa saldo: Rp " . number_format($saldo, 0, ',', '.') . "\n\n";
            } else echo "Saldo tidak cukup!\n\n";
            break;

        case 3:
            echo "Jumlah setor: ";
            $setor = trim(fgets(STDIN));
            $saldo += $setor;
            echo "Setor berhasil. Saldo kini: Rp " . number_format($saldo, 0, ',', '.') . "\n\n";
            break;

        case 4:
            echo "Jumlah transfer: ";
            $trf = trim(fgets(STDIN));
            if ($trf <= $saldo) {
                $saldo -= $trf;
                echo "Transfer berhasil. Sisa saldo: Rp " . number_format($saldo, 0, ',', '.') . "\n\n";
            } else echo "Saldo tidak cukup!\n\n";
            break;

        case 5:
            echo "Terima kasih!\n";
            exit;

        default:
            echo "Menu tidak tersedia!\n\n";
    }
}
?>

File: switch case.py
# ATM Sederhana - Python

# Validasi Nama
while True:
    nama = input("Masukkan Nama: ")
    cek = input("Apakah nama sudah benar? (TRUE/FALSE): ").upper()
    if cek == "TRUE":
        break

# Input NIM → Saldo
nim = input("Masukkan NIM (angka saja): ")
saldo = int(nim)

print("\n=== DATA NASABAH ===")
print(f"Nama  : {nama}")
print(f"Saldo : Rp {saldo:,.0f}\n")

# Menu ATM
while True:
    print("===== MENU ATM =====")
    print("1. Cek Saldo")
    print("2. Tarik Tunai")
    print("3. Setor Tunai")
    print("4. Transfer")
    print("5. Keluar")
    
    pilih = input("Pilih menu: ")

    match pilih:
        case "1":
            print(f"Saldo Anda: Rp {saldo:,.0f}\n")

        case "2":
            tarik = int(input("Jumlah tarik: "))
            if tarik <= saldo:
                saldo -= tarik
                print(f"Tarik berhasil. Sisa saldo: Rp {saldo:,.0f}\n")
            else:
                print("Saldo tidak cukup!\n")

        case "3":
            setor = int(input("Jumlah setor: "))
            saldo += setor
            print(f"Setor berhasil. Saldo kini: Rp {saldo:,.0f}\n")

        case "4":
            trf = int(input("Jumlah transfer: "))
            if trf <= saldo:
                saldo -= trf
                print(f"Transfer berhasil. Sisa saldo: Rp {saldo:,.0f}\n")
            else:
                print("Saldo tidak cukup!\n")

        case "5":
            print("Terima kasih!")
            break

        case _:
            print("Menu tidak tersedia!\n")